src/mono/wasm/runtime-test.js

```diff
@@ -201,9 +201,9 @@ var Module = {
 			}
 		}
 
-		if (enable_gc) {
-			var f = Module.cwrap ('mono_wasm_enable_on_demand_gc', 'void', []);
-			f ();
+		if (!enable_gc) {
+			var f = Module.cwrap ('mono_wasm_enable_on_demand_gc', 'void', ['number']);
+			f (enable_gc);
```

Comment:
this is a bit confusing, we already know that enable_gc can only be 0 in here. I'd prefer keeping the variable as a bool and passing 0 explicitly here.